.github/workflows/publish.yml

```
@@ -13,7 +13,7 @@ jobs:
         uses: actions/setup-java@5ffc13f4174014e2d4d4572b3d74c3fa61aeb2c2 # v3.11.0
         with:
           distribution: 'corretto'
-          java-version: 8
+          java-version: 11
```

@jeromevdl why are we changing this for v1 / `main` ? seems like a potential problem for little gain - we've never run a publish with a different JDK version, and seeing as we're going to redo the publishing process in v2 anyway probably, I don't see much upside in changing this towards the end of the lifecycle of v1.

---

as you prefer. I don't see any risk here but if you want
```suggestion
          java-version: 8
```